Makefile

```diff
@@ -190,6 +190,11 @@ distclean:
 	rm -rf $(DEPS_BUILD_DIR) build
 	$(MAKE) clean
 
+verify-term:
+	@echo "DEBUG: checking current detected term"
+	$(NVIM_PRG) -i NONE --headless -u ci/headless_init.lua +'echo $TERM' +'qall!'
+	# $(NVIM_PRG) -i NONE --headless -u ci/headless_init.lua +'checkheatlh nvim' +'qall!'
```

Comment:
? no

Reply:
just trying to see what happened here: https://builds.sr.ht/~jmk/job/752468#task-functionaltest-1550, which is obviously unrelated to this change..

ironically it's what I'm trying to get around in this PR 😄